fe/fe-core/src/main/java/org/apache/doris/qe/cache/PartitionRange.java

```diff
@@ -48,6 +48,8 @@
 import java.util.List;
 import java.util.Map;
 
+import static org.apache.doris.analysis.DateLiteral.BASIC_ZERO_TIME;
```

Comment:
not using static import:
http://doris.incubator.apache.org/master/zh-CN/developer-guide/java-format-code.html#import-order